raiden-ts/src/raiden.ts

```diff
@@ -28,7 +28,7 @@ import kovanDeploy from './deployment/deployment_kovan.json';
 import goerliDeploy from './deployment/deployment_goerli.json';
 
 import { ContractsInfo, RaidenContracts, RaidenEpicDeps, TokenInfo } from './types';
-import { ShutdownReason } from './constants';
+import { ShutdownReason, MATRIX_KNOWN_SERVERS_URL } from './constants';
```

Comment:
Could you, please, move these constants also to `config.ts`? Maybe directly on a `const defaultConfig: RaidenConfig` export with these values on. Some of the default-constructed config parameters may be dynamic and depend on `create` factory/constructor context, but most of it should be constant and would go well in a single place like `config.ts`